crm_phonecall/views/crm_phonecall_view.xml

```diff
@@ -1,12 +1,11 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
     <record id="crm_phonecall_categ_action" model="ir.actions.act_window">
         <field name="name">Phonecall Categories</field>
         <field name="res_model">crm.lead.tag</field>
         <field name="view_type">form</field>
         <field name="view_id" ref="crm.crm_lead_tag_tree"/>
-        <field name="domain">[]</field>
```

Comment:
Don't remove this one for making sure it adds an empty domain when coming from other versions that can contain other thing.